state/futures/hooks.ts

```diff
@@ -26,7 +26,11 @@ export const usePollFuturesData = () => {
 	const { providerReady } = Connector.useContainer();
 
 	useEffect(() => {
-		if (providerReady) dispatch(fetchCrossMarginSettings());
+		if (providerReady) {
+			dispatch(fetchCrossMarginSettings());
+			dispatch(fetchIsolatedMarginAccountData());
+			dispatch(fetchCrossMarginAccountData());
```

Comment:
@Tburm I forgot to mention that startPolling will also do a fetch before the interval so there is no need to also fetch here.

Reply:
Or actually maybe you did this so it will be fetched always on dashboard. If that's the case I think we should create a dashboard specific hook like this. I can take a look at that.